### ISSUE TITLE
Temperature units for Celsius 

### ISSUE THREAD

**Issue by mdeceglie** [hackingmaterials/pv-terms#20]
The temperature units are currently expressed as `C`, which is technically the SI symbol for coulomb. For parameters like `temperature_cell` Celsius  is natural, so I suggest using °C. For units where temperature appears in the denominator, kelvin may be more appropriate. For example, in [Table 4 on this NIST site](https://physics.nist.gov/cuu/Units/units.html) derived units use kelvin.

**Comment by cwhanse** [comment]
Are you suggesting that the documentation text include the degree symbol, or that the unit be changed to K? I'm not in favor of the K option.

The degree symbol is awkward to type (ALT+0176 on a Windows keyboard) but could be added to the documentation file.

**Comment by mdeceglie** [comment]
Yes, for temperatures such as `temperature_cell` I am suggesting that the documentation text and `definitions.csv` include the degree symbol. The units as they are now, without it, specify coulombs.

For units where the temperature appears in the denominator such as `gamma_pmp` I suggest changing to K, but could go either way on this point as long as the degree symbol is included along with `C`.

**Comment by toddkarin** [comment]
Sounds good to me. For gamma_pmp, do we want W/°C or W/K? I'd vote for W/°C for the engineering crowd.

**Comment by cwhanse** [comment]
We can try adding the degree symbol to a line in definitions.csv and see how it renders. @mdeceglie @toddkarin care to make a PR?

**Comment by mdeceglie** [comment]
I'll give it a shot and open a PR. Thanks!

**Comment by steve-ransome** [comment]
If "°" <alt + 0176> causes problems I've seen degC used before.
